src/snowflake/cli/api/config.py

```diff
@@ -96,16 +111,16 @@ def config_section_exists(*path) -> bool:
 
 def get_connection(connection_name: str) -> dict:
     try:
-        return get_config_section("connections", connection_name)
+        return get_config_section(CONNECTIONS_SECTION, connection_name)
     except KeyError:
         raise MissingConfiguration(f"Connection {connection_name} is not configured")
 
 
 def get_config_section(*path) -> dict:
     section = _find_section(*path)
-    if type(section) is Container:
+    if isinstance(section, Container):
```

Comment:
Previous implementation wasn't covering all types and because of that my local config loading was failing.